html/transaction-history-retry-onRetry.html

```diff
@@ -40,64 +48,64 @@
         <div class="bankHeading">Global eBank</div>
     </div>
     <div class="stepIntro">
-        <div class="stepIntroText">Your Account</div>
+        <div class="stepIntroText" data-externalizedString="YOUR_ACCOUNT"></div>
     </div>
     
     <div id="loader" class="loader"></div>
     <div id="transHistory">
-        <div class="stepIntroText">Recent Transactions</div>
+        <div class="stepIntroText" data-externalizedString="RECENT_TRANSACTIONS"></div>
         <table class="transaction">
             <tr>
-                <th class="tableText" style="width:130px">DESCRIPTION</th>
-                <th class="bl tableText" style="width:70px">TYPE</th>
-                <th class="bl tableText" style="width:70px">AMOUNT</th>
-                <th class="bl tableText" style="width:70px">BALANCE</th>
+                <th class="tableText" style="width:130px" data-externalizedString="DESCRIPTION"></th>
+                <th class="bl tableText" style="width:70px" data-externalizedString="TYPE"></th>
+                <th class="bl tableText" style="width:70px" data-externalizedString="AMOUNT"></th>
+                <th class="bl tableText" style="width:70px" data-externalizedString="BALANCE"></th>
             </tr>
             <tr>
-                <td class="tableText">APRIL 4, 2018</td>
+                <td class="tableText" data-externalizedString="MONTH"> 4, 2018</td>
                 <td></td>
                 <td></td>
                 <td></td>
             </tr>
             <tr>
-                <td class="bl tableText">Credit card autopay</td>
-                <td class="bl tableText">Debit</td>
+                <td class="bl tableText" data-externalizedString="DESC_CREDIT_CARD"></td>
+                <td class="bl tableText" data-externalizedString="TYPE_DEBIT"></td>
                 <td class="bl tableText">-$200</td>
                 <td class="bl tableText">$1785</td>
             </tr>
             <tr>
-                <td class="tableText">APRIL 3, 2018</td>
+                <td class="tableText" data-externalizedString="MONTH"> 3, 2018</td>
                 <td></td>
                 <td></td>
                 <td></td>
             </tr>
             <tr>
-                <td class="bl tableText">Food</td>
-                <td class="bl tableText">Debit</td>
+                <td class="bl tableText" data-externalizedString="DESC_FOOD"></td>
+                <td class="bl tableText" data-externalizedString="TYPE_DEBIT"></td>
                 <td class="bl tableText">-$15</td>
                 <td class="bl tableText">$1985</td>
             </tr>
             <tr>
-                <td class="tableText">APRIL 2, 2018</td>
+                <td class="tableText"data-externalizedString="MONTH"> 2, 2018</td>
```

Comment:
Missing a space before `data...`

This still works in Chrome, but not sure if it might cause problems in older browsers.